src/js/views/LottoContainerView.js

```diff
@@ -0,0 +1,109 @@
+import { EVENT } from '../constants/events';
+import { SELECTOR } from '../constants/selector';
+import { findElement } from '../utils/dom';
+import { emitListener } from '../utils/event';
+
+class LottoContainerView {
+  #app = null;
+
+  #chargeForm = null;
+
+  #alignConverter = null;
+
+  #lottoContainer = null;
+
+  #purchasedMessage = null;
+
+  #alignConverterContainer = null;
+
+  #onSubmitCharge = null;
+
+  #onChangeAlignState = null;
+
+  #onInputOverMaxLength = null;
+
+  constructor({ $app }) {
+    this.#app = $app;
+    this.#initializeTemplate();
+    this.#initializeDOM();
+    this.#bindEventHandler();
+  }
+
+  #initializeTemplate() {
+    this.#app.insertAdjacentHTML('beforeend', this.#basicTemplate);
+  }
+
+  #initializeDOM() {
+    this.#chargeForm = findElement(SELECTOR.CHARGE_INPUT_FORM);
+    this.#alignConverter = findElement(SELECTOR.ALIGN_CONVERTER);
+    this.#lottoContainer = findElement(SELECTOR.LOTTO_CONTAINER);
+    this.#purchasedMessage = findElement(SELECTOR.PURCHASED_MESSAGE);
+    this.#alignConverterContainer = findElement(SELECTOR.ALIGN_CONVERTER_CONTAINER);
+  }
+
+  #bindEventHandler() {
+    this.#onSubmitCharge = (e) => emitListener(EVENT.SUBMIT_CHARGE, e);
+    this.#onChangeAlignState = (e) => emitListener(EVENT.CHANGE_ALIGN_STATE, e);
+    this.#onInputOverMaxLength = (e) => emitListener(EVENT.INPUT_OVER_MAX_LENGTH, e);
+
+    this.#chargeForm.addEventListener('submit', this.#onSubmitCharge);
+    this.#alignConverter.addEventListener('change', this.#onChangeAlignState);
+    this.#chargeForm.addEventListener('input', this.#onInputOverMaxLength);
```

Comment:
```suggestion
    this.#chargeForm.addEventListener('submit', (e) => emitListener(EVENT.SUBMIT_CHARGE, e));
    this.#alignConverter.addEventListener('change', (e) => emitListener(EVENT.CHANGE_ALIGN_STATE, e));
    this.#chargeForm.addEventListener('input', (e) => emitListener(EVENT.INPUT_OVER_MAX_LENGTH, e));
```
#bindEventHandler안에서 이벤트 핸들러로만 쓰인다면 따로 클래스 필드로 선언하지 않아도 될 것 같네요.

Reply:
질문이 있는데 율리의 코드에서 removeEventListener는 어떻게 할 수 있나요?? 

사실 당장 필요한 것이 아니라 고려하지 않았어야하는 생각도 있습니다. 하지만 클래스 필드로 선언한데에는 참조값을 알고 있어야 삭제할 수 있으니까 라는 이유가 있었습니다.

율리가 짜주신 방식이 기존의 코드이고, 삭제 기능을 위해 클래스 필드를 사용한 것이 리팩토링 과정에서 추가된 부분인데, 어떻게 생각하시나요?

Reply:
아- removeEventListener를 고려한거다. 그럼 이해됩니다~ 그럼 removeEventListener를 바인딩하는 코드도 있는게 좋겠죠? 지금 당장 필요하지 않는다면 필요한 시점에 리팩토링하는건 어떨 까요?

Reply:
맞는 것 같아요 !!! `필요할 때 작성한다` 라는 사실을 지향하며 코드를 수정해보겠습니다.